packages/datadog-plugin-jest/test/index.spec.js

```diff
@@ -198,17 +204,20 @@ describe('Plugin', () => {
         }
         datadogJestEnv.handleTestEvent(testEvent)
         expect(tracer._tracer.wrap).to.have.been.called
+        tracer._tracer.wrap = originalWrap
       })
 
       it('should not call wrap on events other than test_start or test_skip', () => {
         if (process.env.DD_CONTEXT_PROPAGATION === 'false') return
+        const originalWrap = tracer._tracer.wrap
         tracer._tracer.wrap = sinon.spy(() => {})
 
         const testFnStartEvent = {
           name: 'test_fn_start'
         }
         datadogJestEnv.handleTestEvent(testFnStartEvent)
         expect(tracer._tracer.wrap).not.to.have.been.called
+        tracer._tracer.wrap = originalWrap
```

Comment:
unrelated fix

Reply:
We should probably figure out a way to not do that as soon as possible since it makes assumption on the tracer internals.

Reply:
I just created a jira issue to track it